src/agent/src/agent_registration.cpp

```diff
@@ -41,16 +45,12 @@ namespace agent_registration
 
         if (!token.has_value())
         {
-            LogError("Failed to authenticate with the manager");
+            std::cout << fmt::format("Failed to authenticate with the manager\n");
```

Comment:
This doesn't seem to need `fmt::format`.

Reply:
fixed!